app/dashboard/src/views/dashboard/member/edit/EditMemberRecordsView.vue

```diff
@@ -286,7 +286,10 @@ export default class EditMemberRecordsView extends Mixins(NavigationMixin) {
     set allowData(enabled: boolean) { this.setBooleanType(RecordType.NoData, !enabled) }
 
     get allowPictures() { return !this.getBooleanType(RecordType.NoPictures) }
-    set allowPictures(enabled: boolean) { this.setBooleanType(RecordType.NoPictures, !enabled) }
+    set allowPictures(enabled: boolean) { this.setBooleanType(RecordType.NoPictures, !enabled)} 
+    
+    get allowOnlyGroupPictures() { return !this.getBooleanType(RecordType.OnlyGroupPictures) }
+    set allowOnlyGroupPictures(enabled: boolean) { this.setBooleanType(RecordType.OnlyGroupPictures, !enabled)}
```

Comment:
Denk dat hier telkens het omgekeerde wordt teruggegeven en ingesteld. 
allowOnlyGroupPictures returnt true als OnlyGroupPictures afwezig is in de records